synapse/lib/layer.py

```diff
@@ -300,10 +300,10 @@ def __init__(self, layr, form, prop):
         self.prop = prop
 
     def getNodeValu(self, buid):
-        bkey = buid + b'\x01' + self.prop.encode()
-        byts = self.layr.layrslab.get(bkey, db=self.layr.bybuid)
-        if byts is not None:
-            return s_msgpack.un(byts)[0]
+        sode = self.layr._getStorNode(buid)
+        valt = sode['props'].get(self.prop)
```

Comment:
can raise keyerror

Reply:
actually, it can't.  it's a default dict.